code/obj/machinery/computer/card.dm

```diff
@@ -38,6 +38,14 @@
 	icon = 'icons/obj/computerpanel.dmi'
 	icon_state = "id2"
 
+/obj/machinery/computer/card/portable
```

Comment:
Probably missing a few things on this. Even though it looks like a briefcase it's still is anchored and blocks movement, as far as I can tell.

Reply:
the personal laptop also blocks movement, idk there's a kinda precedent for this happening

Reply:
ya all the luggable computers do that

Reply:
movement blockage though no, ill look at that

Reply:
found it, density = 0

Reply:
so now, TLDR its still anchored (as all the other computers of this variety are) but now allows you to move through it